### ISSUE TITLE
Support hostType lan

### ISSUE THREAD

**Issue by illyax** [expo/detox-expo-helpers#6]
I noticed detox with expo works only with hostType tunnel, which is slower than lan,
any chance you will support lan?

**Comment by crearc** [comment]
Yeah this really stinks... I also didn't realize it wasn't blacklisting everything else. It's unfortunately not being supported.
